CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Unreleased
```

Comment:
I would prefer having an "unreleased full changelog" as a placeholder, so people don't forget to add it when they start adding things to the changelog.